CoreScriptsRoot/Modules/Panel3D.lua

```diff
@@ -250,40 +253,48 @@ function Panel3D.GetGUI(panel)
 end
 
 local zeroVector = Vector3.new(0, 0, 0)
-local baseHorizontal = CFrame.new()
-local basePosition = Vector3.new()
+local headHorizontal = CFrame.new()
+local headOffset = Vector3.new()
 local currentHoverPanel = nil
 local savedMouseBehavior = Enum.MouseBehavior.Default
 function Panel3D.OnRenderStep()
 	if not UserInputService.VREnabled then
 		return
 	end
+	local cameraCFrame = workspace.CurrentCamera.CFrame
 	local cameraRenderCFrame = workspace.CurrentCamera:GetRenderCFrame()
 	local userHeadCFrame = UserInputService:GetUserCFrame(Enum.UserCFrame.Head)
-	local cameraLook = cameraRenderCFrame.lookVector
+
+	local cameraLook = cameraCFrame.lookVector
 	local cameraHorizontalVector = Vector3.new(cameraLook.X, 0, cameraLook.Z).unit
-	local cameraPitchAngle = math.asin(cameraLook.Y)
+	local cameraHorizontal = CFrame.new(zeroVector, cameraHorizontalVector)
```

Comment:
`cameraHorizontal`, `cameraHorizontalVector`, `cameraLook` don't seem to be used outside of referencing each other, can we remove them?